nautobot/ipam/forms.py

```diff
@@ -751,7 +749,7 @@ class IPAddressBulkCreateForm(BootstrapMixin, forms.Form):
     pattern = ExpandableIPAddressField(label="Address pattern")
 
 
-class IPAddressBulkAddForm(BootstrapMixin, TenancyForm, AddressFieldMixin, CustomFieldModelForm):
+class IPAddressBulkAddForm(NautobotModelForm, TenancyForm, AddressFieldMixin):
```

Comment:
Not sure about this change - this is a special bulk-add form, are you confident that it works correctly when inheriting from NautobotModelForm?

Reply:
It was an migration issue, and after fixing it, inserting `NautobotModelForm `and replacing the other two, also works for `IPAddressBulkAddForm`.